README.md

```diff
@@ -238,38 +240,27 @@ a single failure.
 
 To learn more about segments, see the [Corfu wiki](https://github.com/CorfuDB/CorfuDB/wiki).
 
-To scale Corfu, we add additional ``stripes''. To add an additional stripe, first 
-start a new ```corfu_server``` on port 9002:
+To add more nodes, begin by 
+starting a new ```corfu_server``` on port 9002:
 ```
 ./CorfuDB/bin/corfu_server -m 9002
 ```
```

Comment:
I think it would be cleaner to just use `/bin/corfu_remove_node` and assume that the user is in the parent directory.

Reply:
this is done for consistency with the rest of the command examples, which are all in the parent dir of CorfuDB

Reply:
I meant change all of them.